CHANGELOG.md

```diff
@@ -14,6 +14,15 @@ Releases 4.5.x and 4.6.x are based on mainline support submitted in
 
 ## [Unreleased]
 
+## [v4.6.9] - 2018-05-11
+### Added
+- IOMMU support enabled, but not yet stable- [tl;dr](https://github.com/pcengines/apu2-documentation/tree/master/docs/iommu.md)
```

Comment:
@miczyg1 let's not add it in this release

BTW looks like we don't have much items added in this release

Reply:
Well I have worked pretty hard on USB, that consumed huge amounts of time.

Reply:
@pietrushnic wait for a while with merging, I will push a fix for serial number too